packages/build/src/log/messages/compatibility.js

```diff
@@ -5,9 +5,20 @@ import { getPluginOrigin } from '../description.js'
 import { logArray, logSubHeader, logWarningArray, logWarningSubHeader } from '../logger.js'
 import { THEME } from '../theme.js'
 
+export const logRuntime = (logs, pluginOptions) => {
+  const runtimes = pluginOptions.filter(isRuntime)
+
+  // Once we have more runtimes, this harcoded check should be removed
+  if (runtimes.length > 1) {
+    logSubHeader(logs, 'Using Next.JS Runtime')
```

Comment:
Can you get this from the plugin name?

Reply:
Maybe? checking the manifest and the package.json I couldn't see anything that is friendly for this format in the requirements. Potentially use the package description.

Or rather, are you suggesting `if (runtimes.includes('@netlify/next-runtime') {...` instead?

Reply:
Ah I was thinking they were plugin info objects, which would include the name. If they're just package names then sure, hard-code it. But fix the casing: 
```suggestion
    logSubHeader(logs, 'Using Next.js Runtime')
```

Reply:
Yeah as far as I can tell higher up the call chain, nothing of use is populated into this tree. Thanks for the casing spot!